### ISSUE TITLE
Dashboard no longer updating

### ISSUE THREAD

**Issue by dalepotter** [IATI/IATI-Dashboard#408]
The live and dev instances of the Dashboard have not updated since `2016-09-14`.

The output of the logs show:
```
dashboard@webserver11:~/logs$ tail 20160929-dashboard.log -n 50
->  invalidxml
->  nonstandardroots
->  publisher_unique_identifiers
->  validation
->  versions
make_csv.py
speakers kit.py
Unrecognised registry publisher_country code:  
Unrecognised registry publisher_country code:  
make_html.py
Doing initial data import
Initial data import finished
Traceback (most recent call last):
  File "make_html.py", line 312, in <module>
    freezer.freeze()
  File "/home/dashboard/IATI-Dashboard/pyenv/local/lib/python2.7/site-packages/flask_frozen/__init__.py", line 149, in freeze
    new_filename = self._build_one(url)
  File "/home/dashboard/IATI-Dashboard/pyenv/local/lib/python2.7/site-packages/flask_frozen/__init__.py", line 257, in _build_one
    base_url=base_url)
  File "/home/dashboard/IATI-Dashboard/pyenv/local/lib/python2.7/site-packages/werkzeug/test.py", line 762, in get
    return self.open(*args, **kw)
  File "/home/dashboard/IATI-Dashboard/pyenv/local/lib/python2.7/site-packages/flask/testing.py", line 108, in open
    follow_redirects=follow_redirects)
  File "/home/dashboard/IATI-Dashboard/pyenv/local/lib/python2.7/site-packages/werkzeug/test.py", line 736, in open
    response = self.run_wsgi_app(environ, buffered=buffered)
  File "/home/dashboard/IATI-Dashboard/pyenv/local/lib/python2.7/site-packages/werkzeug/test.py", line 659, in run_wsgi_app
    rv = run_wsgi_app(self.application, environ, buffered=buffered)
  File "/home/dashboard/IATI-Dashboard/pyenv/local/lib/python2.7/site-packages/werkzeug/test.py", line 855, in run_wsgi_app
    app_iter = app(environ, start_response)
  File "/home/dashboard/IATI-Dashboard/pyenv/local/lib/python2.7/site-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/dashboard/IATI-Dashboard/pyenv/local/lib/python2.7/site-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/home/dashboard/IATI-Dashboard/pyenv/local/lib/python2.7/site-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/home/dashboard/IATI-Dashboard/pyenv/local/lib/python2.7/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/dashboard/IATI-Dashboard/pyenv/local/lib/python2.7/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/dashboard/IATI-Dashboard/pyenv/local/lib/python2.7/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/dashboard/IATI-Dashboard/pyenv/local/lib/python2.7/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/dashboard/IATI-Dashboard/pyenv/local/lib/python2.7/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "make_html.py", line 217, in publisher
    publisher_licenses=licenses.licenses_for_publisher(publisher),
  File "/home/dashboard/IATI-Dashboard/licenses.py", line 87, in licenses_for_publisher
    return set([ package.get('license_id') for package in ckan[publisher_name].values() ])
KeyError: u'britishcouncil'
```

**Comment by dalepotter** [comment]
This appears to be due to an issue with a publishers dataset containing no data:
![image](https://cloud.githubusercontent.com/assets/8247168/18989568/93eda6f0-8704-11e6-9607-f359a5098659.png)

(In normal cases this would show a big 'Download' button, rather than "This dataset has no data").

This is causing the CKAN API to not return a URL when querying organisation metadata.  Additionally, the publisher does not appear within the full ckan metadata, stored in `ckan.json`.

**Comment by dalepotter** [comment]
Following deployment of the fix in 	d7af111, both the live and dev dashboards have successfully updated.

This issue will now be closed.
